internal/bminventory/inventory.go

```diff
@@ -432,6 +437,9 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		params.NewClusterParams.UserManagedNetworking = swag.Bool(true)
 		// in case of single node VipDhcpAllocation should be always false
 		params.NewClusterParams.VipDhcpAllocation = swag.Bool(false)
+	} else if cpuArchitecture != common.DefaultCPUArchitecture {
+		err = errors.Errorf("Non x86_64 CPU architectures are supported only for single node OpenShift")
```

Comment:
Really?
This is news to me.

Reply:
We can start with that until Ironic is supported for arm, and the x86 baremetal-install is available in the arm release image.

Reply:
Changed the validation to ensure User Managed Networking, as seems none platform multi-node can be indeed installed as well.